frontend/js/views/list-annotation.js

```diff
@@ -581,6 +536,99 @@ define(["jquery",
             remove: function () {
                 this.commentContainer.remove();
                 Backbone.View.prototype.remove.apply(this, arguments);
+            },
+
+            /**
+             * Add a new annotation with a content item containing free text.
+             * @alias module:views-list-annotation.ListAnnotation#addFreetextContent
+             * @param {Event} event Event object
+             */
+            addFreeTextContent(event) {
+                event && event.stopPropagation();
```

Comment:
```suggestion
                event.stopPropagation();
```

Is this ever called without event?